### PR TITLE
update oncall groups in OWNERS_ALIASES

### PR DESCRIPTION
**Pull request by BenTheElder**
/cc @justaugustus @chaodaiG 
xref: https://kubernetes.slack.com/archives/CCK68P2Q2/p1605913365196200?thread_ts=1605906060.178100&cid=CCK68P2Q2

### CODE REVIEW

File: OWNERS_ALIASES [hunk 1]
```diff
@@ -1,14 +1,21 @@
 aliases:
   # business hours oncall for prow etc (https://go.k8s.io/oncall)
   test-infra-oncall:
-  - BenTheElder
+  - chaodaig
```
Comment (chaodaiG) [inline OWNERS_ALIASES:4]: This is a dup, mind delete one or the other?

Comment (chaodaiG) [reply OWNERS_ALIASES:4]: Dup with line https://github.com/kubernetes/test-infra/pull/20007/files#diff-4985b733677adf9dda6b5187397d4700868248ef646d64aecfb66c1ced575499R10

**Comment by BenTheElder** [comment]
d'oh, intent was to sort

**Comment by chaodaiG** [comment]
> d'oh, intent was to sort

Totally get it
/lgtm
/approve

File: OWNERS_ALIASES (hunk 1 above)
Comment (BenTheElder) [reply OWNERS_ALIASES]: fixed

**Pull request merged**